kolibri/plugins/perseus_viewer/assets/src/views/PerseusRendererIndex.vue

```diff
@@ -1,22 +1,16 @@
 <template>
 
-  <div v-if="itemId || itemData" class="bibliotron-exercise perseus-root">
-    <div class="framework-perseus" :class="{ 'perseus-mobile': isMobile }">
-      <div id="perseus" ref="perseus" style="background-color: white;">
-        <div class="loader-container">
-          <KLinearLoader
-            v-show="loading"
-            :delay="false"
-            type="indeterminate"
-          />
-        </div>
-        <div
-          id="problem-area"
-          :dir="contentDirection"
-        >
-          <div id="workarea" style="margin-left: 0px"></div>
-        </div>
-
+  <div
+    v-if="itemId || itemData"
+    class="bibliotron-exercise perseus-root"
+    :class="{ 'perseus-mobile': isMobile }"
+  >
+    <LessonMasteryBar
```

Comment:
I _think_ this is the right place for this - although if it's something we would expose in, say, the QTI renderer too, then we might need to move it out of here in the future.

Reply:
If we want it in the QTI renderer we can move it to Core and update the imports for sure if we keep that part of the design consistent between Perseus & QTI.